src/main/kotlin/fi/metatavu/timebank/api/persistence/model/TimeEntry.kt

```diff
@@ -37,4 +37,20 @@ class TimeEntry {
 
     @Column
     var updatedAt: OffsetDateTime? = null
+
+    /**
+     * Compares object equality ignoring entryId
+     */
+    override fun equals(other: Any?): Boolean {
```

Comment:
Maybe a cleaner version would be to use kotlin reflection library. Which looks something like this:

```
import java.util.*
import kotlin.reflect.full.memberProperties

data class Example(val a: Int, val b:Long, val c: String)

fun areTwoObjectsSame(example: Example) {
    val baseValue = this
    val differentFields = Example::class.memberProperties.filter {
        val firstValue = it.get(baseValue)
        val secondValue = it.get(example)
        !Objects.equals(firstValue, secondValue)
    }

    return differentFields.size == 0
}
```